vue/components/ui/molecules/form-input/form-input.vue

```diff
@@ -53,6 +48,12 @@
     flex: 1;
 }
 
+.form-input .tooltip > .header {
+    -webkit-text-decoration: underline #57626e dashed;

```

Comment:
no need to use the `-webkit` prefixed values

Reply:
@joaofsl Mentioned this as well, but in this case, if this property isn't here, Safari shows no text-decoration at all. Shouldn't this be preserved to maximise browser compatibility?